src/scripts/components/VideoForm.js

```diff
@@ -118,6 +123,30 @@ class VideoForm extends Component<Props, Object> {
       title: nextProps.selectedVideo.title,
       description: nextProps.selectedVideo.description
     })
+
```

Comment:
I save in the component state the value coming form the global state. Don't know if it's a bad approach..